eng/source-build-patches/0008-use-sourcebuildRID-to-find-clrjit.patch

```diff
@@ -1,25 +0,0 @@
-From 2ac4c152b5c4409025dcbaf473b89a6094453e97 Mon Sep 17 00:00:00 2001
```

Comment:
TODO: double check binlog to ensure this patch isn't needed.

Reply:
I checked the binlog and it's set to the correct value here so the patch isn't needed. 
![image](https://user-images.githubusercontent.com/2030323/118736791-3f7f1e80-b7f8-11eb-9884-80d8e766c12e.png)

Looking through the msbuild output, I see that in https://github.com/dotnet/aspnetcore/blob/8b52d1c8f631051a6e03aea4a4ec9f808d998cff/eng/SourceBuild.props#L65 `SourceBuildRuntimeIdentifier` is always set to `TargetRuntimeIdentifier` which means the logic in this patch is no longer needed. Could the value of `SourceBuildRuntimeIdentifier` be different from `TargetRuntimeIdentifier`? If so, then we still need the logic in this patch.

Reply:
@MichaelSimons can you answer the question:
>  However, could the value of SourceBuildRuntimeIdentifier be different from TargetRuntimeIdentifier, i.e. overridden somewhere else?

Reply:
@MichaelSimons bump. Can you confirm whether we need to account for scenarios where `SourceBuildRuntimeIdentifier` and `TargetRuntimeIdentifier` are different?

Reply:
Discussing this with @dseefeld, this was originally added in order to support portable builds.  We should eliminate this patch as we are not certain these patches are necessary now.